install_files/ansible-base/roles/app-test/tasks/disable_mprotect_ff_plugin_container.yml

```diff
@@ -8,7 +8,6 @@
   # We'll catch that error and respond accordingly in the next task.
   failed_when: false
   register: paxctl_firefox_header_check
-  when: ansible_kernel.endswith('-grsec')
```

Comment:
Why is it necessary to remove this? (while the app-test role is used in staging, which does use grsec kernels, it doesn't hurt to leave this in explicitly no? Or did this produce some other issue?)

Reply:
Ahh I see. For other reviewers, the `when` condition was duplicated (see below)